libs/components/core/src/lib/modules/shared/number-format/number-format-utility.ts

```diff
@@ -32,7 +32,7 @@ export class SkyNumberFormatUtility {
     style: SkyIntlNumberFormatStyle,
     digits?: string | null,
     currency: string | null = null,
-    currencyAsSymbol = false,
+    currencyDisplay: 'code' | 'symbol' | 'narrowSymbol' | 'name' = 'symbol',
```

Comment:
I'm a bit surprised this isn't failing any unit tests. The percent pipe does not send a default for `currencyAsSymbol` which would have resulted in `code` under the current code. But with this - the percent pipe would now end up with `symbol` if I'm reading everything correctly. I would like to maintain the current defaults if possible (where the percent pipe uses `code` and the numeric service defaults to `symbol`.

Reply:
The reason all the tests show it working as expected is because currency display options don't impact the percent pipe. But I can see the benefit in drawing the consistency a bit tighter in the code. I'll make a couple of tweaks.